BitPayLib/class-bitpaypages.php

```diff
@@ -39,7 +39,7 @@ public function checkout_thank_you( int $order_id ): void {
 		$invoice_id = $_COOKIE['bitpay-invoice-id'] ?? null; // phpcs:ignore
 
 		wp_enqueue_script( 'remote-bitpay-js', $js_script, null, null, false ); // phpcs:ignore
-		wp_enqueue_script( 'bitpay_thank_you', plugins_url( '../js/bitpay_thank_you.js', __FILE__ ), null, 1, false );
+		wp_enqueue_script( 'bitpay_thank_you', plugins_url( '../../js/bitpay_thank_you.js', __FILE__ ), null, 1, false );
```

Comment:
It hasn't been historically, but we should set the `$ver` param to the version of the plugin. This helps with CDN caching, because it will append a `?ver=5.4.0` to the URL, and CDNs are often set up to treat query params as separate files.

Let's do this for all instances of `wp_enqueue_script()`.

More info: https://developer.wordpress.org/reference/functions/wp_enqueue_script/

> **$ver** `string|bool|null` optional
>
> String specifying script version number, if it has one, which is added to the URL as a query string for cache busting purposes. If version is set to false, a version number is automatically added equal to current installed WordPress version.
> If set to null, no version is added.

Reply:
done